mod-finance/examples/transaction_encumbrance.sample

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "b7553168-60af-4e6d-bcd3-078199a0b353",
```

Comment:
I believe valid example for encumbrance-transaction must contain [encumbrance](https://github.com/folio-org/acq-models/blob/master/mod-finance/schemas/encumbrance.json) sub-object